objects-arrays.js

```diff
@@ -0,0 +1,272 @@
+// ==== Objects ====
+
+/* 
+  Given the following information about dinosaurs, create 3 objects: 
+  Use this pattern to create your objects: 
+  object name, diet, weight, length, period
+*/
+
+// tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceous
+
+const dinosaur1 = {
+  objectname: "tyrannosaurus",
+  diet: "carnivorous",
+  weight: "7000kg",
+  length: "12m",
+  period: "Late Cretaceous"
+};
+
+// stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
+
+const dinosaur2 = {
+  objectname: "stegosaurus",
+  diet: "herbivorous",
+  weight: "2000kg",
+  length: "9m",
+  period: "Late Jurassic"
+};
+
+// velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceous
+
+const dinosaur3 = {
+  objectname: "velociraptor",
+  diet: "carnivorous",
+  weight: "15kg",
+  length: "1.8m",
+  period: "Late Cretaceous"
+};
+
+// Using your dinosaur objects, log answers to these questions:
+
+// How much did tyrannosaurus weigh?
+console.log(dinosaur1["weight"]);
+
+// What was the diet of a velociraptor?
+console.log(dinosaur3["diet"]);
+
+// How long was a stegosaurus?
+console.log(dinosaur2["length"]);
+
+// What time period did tyrannosaurus live in?
+console.log(dinosaur1["period"]);
+
+// Create a new roar method for the tyrannosaurus.  When called, return "RAWERSRARARWERSARARARRRR!" Log the result.
+function roar(tyran) {
+  console.log(`${tyran} "RAWERSRARARWERSARARARRRR!"`);
+}
```

Comment:
For this to be a `method` instead of a regular function, you would define this on the `dinosaur1` object. Either on the object itself or inline:
```js
dinosaur1.roar = () => 'RAWERSRARARWERSARARARRRR';
```